### ISSUE TITLE
Kotlin (KMP) support

### ISSUE THREAD

**Issue by hukpavlo** [thewca/tnoodle-lib#30]
As a developer, I'd like to use tnoodle-lib for web/desktop/iOS platforms.
I think it's possible to rewrite it using Kotlin multiplatform. Any thoughts on this?

**Comment by ndrchvzz** [comment]
Yes it's possible. I rewrote the min2phase bit of tnoodle-lib in Kotlin. I don't think it helps much. I did it mainly as a challenge, and I liked the idea of killing all the java code and have only Kotlin. I haven't published the code yet but I'm planning to do that soon.
As far as I can tell it's open source and there are no license issue with rewriting it in any language you like. You should probably give the author credit for the original code out of respect whenever there are complicated algorithms involved, like in the case of min2phase.
